cmd/kube-apiserver/app/server.go

```diff
@@ -212,7 +213,8 @@ func Run(s *options.APIServer) error {
 		}
 
 		// Set up the tunneler
-		tunneler = master.NewSSHTunneler(s.SSHUser, s.SSHKeyfile, installSSH)
+		healthCheckPath := fmt.Sprintf("http://127.0.0.1:%d/healthz", ports.KubeletReadOnlyPort)
```

Comment:
You would only allow tunneling to nodes that surface unauthenticated ports? That seems… not ideal